helplink.json

```diff
@@ -22,6 +22,7 @@
     "org.worldwideworkshop.olpc.JigsawPuzzle": "jigsaw.html",
     "org.laptop.sugar.Jukebox": "jukebox.html",
     "org.gnome.Labyrinth": "labyrinth.html",
+    "org.sugarlabs.LocoSugar":"locosugar.html",
```

Comment:
Missing whitespace; different style to surrounding lines.

Reply:
fixed. thanks!